release/staging-build.yml

```diff
@@ -124,6 +124,12 @@ batch:
         variables:
           PROJECT_PATH: projects/distribution/distribution
           CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/distribution.distribution
+    - identifier: emissary_ingress_emissary
```

Comment:
Since this isnt in the eks-a bundle/staging build, maybe we could not include it in the staging build? thought @abhay-krishna ?  If we want to exclude we can just add `EXCLUDE_FROM_STAGING_BUILDSPEC=true` to the makefile.

Reply:
Ok I'll try that, it was failing the presubmit if it wasn't included in this, but I prefer the exclude.